### PR TITLE
Add "Intro to ETL" training guide

### PR DESCRIPTION
**Pull request by shaunagm**

### CODE REVIEW

File: docs/training_guides/etl_best_practices.rst [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+Have you ever tried to get data from one platform or system to another? Doing so can be quite a pain, especially if you have to manually download and upload individual files. Luckily, websites and apps often have APIs, which stands for application programming interfaces. These are gateways that let us move data into and out of a system using code.
+
+**ETL**, which stands for Extract, Transform, and Load is the process by which we extract data from one systems API, transform that data so that it’s compatible with a second system's API, and then loading that data into that second system. 
+
+The format that source system gives us data is often different from the format that another system likes receiving data! Everyone thinks their way is best.
+Not so good for the movement, good for data engineers' job security. ;)
+
+Parsons can help us with every step of the way. We can use parsons to extract, transform, and load data! 
+
+Today we're going to be using a Parsons script to move data from Mobilize to Github. Our inspiration was an experience Shauna had managing canvassing volunteers who had signed up on Mobilize, but whose canvassing status was being tracked via Google Sheets.
```
Comment (cmdelrio) [inline docs/training_guides/etl_best_practices.rst:31]: I think you meant Mobilize to Google Sheets ;)

File: docs/training_guides/etl_best_practices.rst [hunk 2]
```diff
@@ -0,0 +1,558 @@
...
+(It's actually something called a `unix timestamp <https://www.unixtimestamp.com/>`_, which measures the total number of seconds since January 1st, 1970. Why January 1st, 1970? No real reason! They just needed to pick a date and I guess that seemed like a good one.)
+
+Let's convert these unix timestamps to something more readable. To do this, we define a function that takes in a value and returns a value::
+
+    def convert_to_legible_date(unix_date):
+        return datetime.utcfromtimestamp(int(unix_date)).strftime('%Y-%m-%d %H:%M:%S')
+
+Here, we're using the ``datetime`` library mentioned above. The ``strftime`` method is what determines the new format. For example, ``%Y`` means "Year with century as a decimal number" (like, say, 1970), and ``%m`` means "Month as a zero-padded decimal number" (like, say, 01). Here's a `cheatsheet <https://strftime.org/>`_ in case you want to play around with the formatting.
+
+Once we've got our function, we can apply it to all the rows in a column by using the `convert_column` function::
```
Comment (cmdelrio) [inline docs/training_guides/etl_best_practices.rst:186]: Maybe here you could say, "by using the parsons table 'convert_column' function" just so it's clear it's a parsons method operating on the parsons table.

**Review by cmdelrio — APPROVED** [review]
This is awesome! It's so clear and oriented towards beginner programers. There were two places with typos or things I thought could be rephrased for clarity.

File: docs/training_guides/etl_best_practices.rst (hunk 1 above)
Comment (shaunagm) [reply docs/training_guides/etl_best_practices.rst]: Good catch!

**Pull request merged**